bundles/binding/org.openhab.binding.ihc/src/main/java/org/openhab/binding/ihc/internal/IhcGenericBindingProvider.java

```diff
@@ -169,10 +276,25 @@ public int getRefreshInterval(String itemName) {
 	}
 
 	@Override
-	public boolean isOutBindingOnly(String itemName) {
+	public boolean isOutBinding(String itemName, int resourceId) {
 		IhcBindingConfig config = (IhcBindingConfig) bindingConfigs
 				.get(itemName);
-		return config != null ? config.outBindingOnly : null;
+		
+		boolean isOutBinding = true;
+		
+		if(config.resourceId == resourceId) {
+			isOutBinding = false;
+		}
+		
+		
+		return isOutBinding;
+	}
+	
+	@Override
+	public boolean hasInBinding(String itemName) {
+		IhcBindingConfig config = (IhcBindingConfig) bindingConfigs.get(itemName);
+		
+		return config != null ? config.resourceId > 0 : null;
```

Comment:
I changed it in that way, that the "In-Binding" will use the resourceId from IhcBindingConfig. Every Out-Going Binding ("1234", ">1234") will use the IhcOutCommandConfig and will get an entry in the outCommandMap.
An out-only binding does not set the resourceId in IhcBindingConfig and therefore it is assigned with zero and hasInBinding would return false.